04/app.js

```diff
@@ -3,17 +3,27 @@ import ReactDOM from 'react-dom';
 
 import File from './File';
 import List from './List';
+import {v4 as uuid} from 'uuid';
 
 class App extends React.Component {
     state = {
         filesList: [],
     }
 
+    handleFile = (file) => {
+        this.setState((state) => {
+            return {
+                filesList: [...state.filesList, file]
+            }
+        })
+    }
+    
+
     render() {
         return (
             <section>
-                <File />
-                <List />
+                <File getFile={this.handleFile}/>
```

Comment:
Ten kod oznacza, że wewnątrz komponentu `<File>` masz dostęp do metody `this.handleFile` poprzez `this.props.getFile`.